docs/releases/1.9.txt

```diff
@@ -356,6 +356,14 @@ Backwards incompatible changes in 1.9
     deprecation timeline for a given feature, its removal may appear as a
     backwards incompatible change.
 
+Paginator.page_range
```

Comment:
If I'm looking at the correct commit 2d542bf60c, it looks like `page_range` was consistently a list in Django 1.8.